src/_pytest/config/__init__.py

```diff
@@ -1953,6 +1953,11 @@ def parse_warning_filter(
             ) from None
     else:
         lineno = 0
+    try:
+        re.compile(message)
+        re.compile(module)
+    except re.error as e:
+        raise UsageError(error_template.format(error=f"invalid regex: {e}"))
```

Comment:
```suggestion
        raise UsageError(error_template.format(error=f"Invalid regex {e.pattern!r}: {e}"))
```

Maybe, so it's clearer for more complex definitions which one of the two is the culprit?

Reply:
sure will update the same, thanks!